build/common.project.props

```diff
@@ -82,7 +82,7 @@
     <Features>strict</Features>
     <!-- Same as SDK default, but without CandidateAssemblyFiles in front, which would search in Content items -->
     <AssemblySearchPaths>{HintPathFromItem};{TargetFrameworkDirectory};{RawFileName}</AssemblySearchPaths>
-    <LangVersion>11</LangVersion>
+    <LangVersion>12</LangVersion>
```

Comment:
This change, and others that GitHub are now showing, are not new in this PR.  I don't know why GitHub thinks it's in the diff, but this is an example of why I never merge dev to my feature branches, and always rebase my feature branches (and therefore force push) instead.

Please fix the branch, so I can see what your changes are.

Reply:
I thought I did rebase form the IDE but I guess not? anyway fixed